### PR TITLE
Installation instructions for Fission Workflows

### PR DESCRIPTION
**Pull request by erwinvaneyk**
This PR adds installation instructions for Fission Workflows 0.2.0 to the documentation website. It is a lightly adapted version of the installation instructions on the Github of Fission Workflows. Most notably, it contains an explanation of the example (which includes some explanation about Fission Workflows)

---
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/453)

### CODE REVIEW

File: Documentation/docs-site/content/install-workflows.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+title: "Fission Workflows Installation Guide"
+date: 2018-01-22T16:03:27.761Z 
+draft: false
+---
+
+
+### Prerequisites
+
+Fission Workflows requires the following components to be installed on your host machine:
```
Comment (soamvasani) [inline Documentation/docs-site/content/install-workflows.md]: I think "host machine" is unclear; also, only installation requires these components.

File: Documentation/docs-site/content/install-workflows.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  WhaleWithFortune:
+    run: whalesay
+    inputs: "{$.Tasks.GenerateFortune.Output}"
+    requires:
+    - GenerateFortune
+```
+
+What you see is the [YAML](http://yaml.org/)-based workflow definition of the `fortunewhale` workflow.
+A workflow consists out of multiple tasks, which are steps that it needs to complete.
+Each task consists out of a unique identifier, such as `GenerateFortune`, a reference to a (Fission) function in the `run` field.
```
[the remark this answers is not in the document]
Comment (life1347) [reply Documentation/docs-site/content/install-workflows.md]: consists of

**Comment by erwinvaneyk** [comment]
@life1347 @soamvasani - addressed your comments, please re-review :)

File: Documentation/docs-site/content/install-workflows.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+```
+
+What you see is the [YAML](http://yaml.org/)-based workflow definition of the `fortunewhale` workflow.
+A workflow consists of multiple tasks, which are steps that it needs to complete.
+Each task has a unique identifier, such as `GenerateFortune`, a reference to a Fission function in the `run` field.
+Optionally, it can contain `inputs` which allows you to specify inputs to the task, 
+as well as contain `requires` which allows you to specify which tasks need to complete before this task can start.
+Finally, at the top you will find the `output` field, which specifies the task whose output is used as the workflow's output. 
+
+In this case, the `fortunewhale` workflow consists out of a sequence of 3 tasks:
```
Comment (life1347) [inline Documentation/docs-site/content/install-workflows.md]: consists of

**Comment by life1347** [comment]
BTW, could you help to check the confliction? Thanks

**Pull request merged**